internal/acceptance/default_namespace_test.go

```diff
@@ -0,0 +1,71 @@
+package acceptance
+
+import (
+	"context"
+	"errors"
+	"testing"
+
+	"github.com/databricks/databricks-sdk-go/apierr"
+	"github.com/databricks/databricks-sdk-go/service/settings"
+	"github.com/databricks/terraform-provider-databricks/common"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestAccDefaultNamespaceSetting(t *testing.T) {
+	t.Skip("Automatic test disabled until feature is enabled")
+	workspaceLevel(t, step{
+		Template: `
+		resource "databricks_default_namespace_settings" "this" {
+			namespace {
+				value = "namespace_value"
+			}
+		}
+		`,
+		Check: resourceCheck("databricks_default_namespace_settings.this", func(ctx context.Context, client *common.DatabricksClient, id string) error {
+			ctx = context.WithValue(ctx, common.Api, common.API_2_1)
+			w, err := client.WorkspaceClient()
+			assert.NoError(t, err)
+			res, err := w.Settings.ReadDefaultWorkspaceNamespace(ctx, settings.ReadDefaultWorkspaceNamespaceRequest{
+				Etag: id,
+			})
+			assert.NoError(t, err)
+			assert.Equal(t, res.Namespace.Value, "namespace_value")
```

Comment:
nit: what does this assert check? Is this necessary?

Reply:
Line 31 checks that the resource exists and can be loaded. Line 32 checks the value of the resource (i.e., that it has been created correctly)